rest-service/manager_rest/resource_manager.py

```diff
@@ -1300,6 +1300,7 @@ def add_node_instance_from_dict(self, instance_dict):
         tenant_name = instance_dict.pop('tenant_name')
         created_by = instance_dict.pop('created_by')
         resource_availability = instance_dict.pop('resource_availability')
+        private_resource = instance_dict.pop('private_resource')
```

Comment:
Won't it fail now that you've removed the attribute from the tables?

Reply:
resource_availability is also not in the table, but it is in the response